docs-v2/integrations/all/wildix.mdx

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Also, this page needs to be referenced in the `mint.json` file.

Reply:
Added this page to all list and link to integration-templates/wildix-pbx